test.js

```diff
@@ -331,3 +331,9 @@ test('doesn\'t write to disk upon instanciation if and only if the store didn\'t
 	exists = fs.existsSync(conf.path);
 	t.is(exists, true);
 });
+
+test('`validate` option', t => {
+	fs.writeFileSync(t.context.conf.path, 'Some text');
```

Comment:
You cannot use the auto-generated instance here. The `.path` is different for each instance.